test/Tests/src/Semantic/Warnings_Spec.enso

```diff
@@ -157,4 +157,13 @@ spec = Test.group "Dataflow Warnings" <|
         result = x.is_static_nothing x
         result . should_equal False
 
+    Test.specify "should be allowed in Vector" <|
+        a = Warning.attach "a" 1
+        b = Warning.attach "b" 2
+        c = [a, b]
+        d = Warning.attach "d" c
+        c . should_equal [1, 2]
+        Warning.get_all c . map .value . should_equal ["b", "a"]
+        Warning.get_all d . map .value . should_equal ["d", "b", "a"]
```

Comment:
Is the order really preserved? Does it have to be? Shouldn't the test just verify the right warnings are present, but ignore the order?

Reply:
I thought the same but then found this: https://github.com/enso-org/enso/blob/develop/distribution/lib/Standard/Base/0.0.0-dev/src/Warning.enso#L30.
Apparently it is fixed to be reverse-chronological.